Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,66 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Need to refactor the code I made in Lesson 2.
+            // Use functions to achieve DRY (as little duplicate code as possible).
+
+            // Name input
```

Comment:
The comments are now redundant, because the function speaks for itself.

Reply:
I have removed the comments as suggested.